scripts/capture-lighthouse.js

```diff
@@ -58,38 +74,36 @@ const lh = async (data) => {
   };
 
   axios.post('https://lighthouse-dot-webdotdevsite.appspot.com//lh/newaudit', d, o)
-    .then(function(response) {
+    .then(function (response) {
       // handle success
       if (response.status === 200 || response.status === 201) {
         if (response.data && response.data.lhr) {
+          const lightHouseData = {};
           const out = response.data.lhr;
 
-          if (!lightHouseData.hasOwnProperty([`${themeKey}`])) {
-            carbonVal = out.audits['resource-summary'].details.items[0].size / 1024 / 1024 / 1024 * 0.06 * 1000
-            lightHouseData[`${themeKey}`] = {
-              performance: Math.ceil(out.categories.performance.score * 100),
-              firstContentfulPaint: Math.ceil(out.audits.metrics.details.items[0].firstContentfulPaint / 100) / 10,
-              firstMeaningfulPaint: Math.ceil(out.audits.metrics.details.items[0].firstMeaningfulPaint / 100) / 10,
-              firstCPUIdle: Math.ceil(out.audits.metrics.details.items[0].firstCPUIdle / 100) / 10,
-              interactive: Math.ceil(out.audits.metrics.details.items[0].interactive / 100) / 10,
-              bestPractices: Math.ceil(out.categories['best-practices'].score * 100),
-              accessibility: Math.ceil(out.categories.accessibility.score * 100),
-              seo: Math.ceil(out.categories.seo.score * 100),
-              carbon: carbonVal.toFixed(3),
-            };
+          carbonVal = out.audits['resource-summary'].details.items[0].size / 1024 / 1024 / 1024 * 0.06 * 1000
+          lightHouseData[`${themeKey}`] = {
+            performance: Math.ceil(out.categories.performance.score * 100),
+            firstContentfulPaint: Math.ceil(out.audits.metrics.details.items[0].firstContentfulPaint / 100) / 10,
+            firstMeaningfulPaint: Math.ceil(out.audits.metrics.details.items[0].firstMeaningfulPaint / 100) / 10,
+            firstCPUIdle: Math.ceil(out.audits.metrics.details.items[0].firstCPUIdle / 100) / 10,
+            interactive: Math.ceil(out.audits.metrics.details.items[0].interactive / 100) / 10,
+            bestPractices: Math.ceil(out.categories['best-practices'].score * 100),
+            accessibility: Math.ceil(out.categories.accessibility.score * 100),
+            seo: Math.ceil(out.categories.seo.score * 100),
+            carbon: carbonVal.toFixed(3),
           }
 
-          fs.writeFileSync(`${path.join(__dirname, '../data')}/themes.json`, JSON.stringify(lightHouseData));
-
+          fs.writeFileSync(dataFile, JSON.stringify(lightHouseData));
         }
       }
 
     })
-    .catch(function(error) {
+    .catch(function (error) {
       // handle error
-      console.log(error);
+      console.log(d);
```

Comment:
`console.log(error)`

Reply:
`d` will log eg
```js
{"url":"https://demo.joomlashine.com/joomla-templates/jsn_ares_2/pro/","replace":true,"save":false}
```
`error` will log something cryptic from axios. I think is better to know which url failed rather that there was a timeout or whatever...
We could log both...

Reply:
Ahh sorry, I didn't see the definition of `d`. Nice variable name btw ;)

Reply:
`d` is the code for `inputData`. Remove the `input` and scrap the `ata`... 😂